akka-stream/src/main/scala/akka/stream/javadsl/StreamConverters.scala

```diff
@@ -89,8 +89,9 @@ object StreamConverters {
 
   /**
    * Creates a Source from an [[java.io.InputStream]] created by the given function.
-   * Emitted elements are `chunkSize` sized [[akka.util.ByteString]] elements,
-   * except the final element, which will be up to `chunkSize` in size.
+   * Emitted elements are up to `chunkSize` sized [[akka.util.ByteString]] elements.
+   * The actual size of the emitted elements depends on how much the incoming stream
+   * delivers on each invocation of [[read()]].
```

Comment:
pretty sure this syntax results in a warning and broken link. `[[]]` is used to generate links, and here scaladoc has no idea what "read" is.